### PR TITLE
Adds a tool to transform Genome-Strip vcf calls into segments.

### PR DESCRIPTION
**Pull request by vruano**

### CODE REVIEW

**Comment by vruano** [comment]
test are failing only due to a separate issue in master described in #415. 

@kareembroad please review.

**Comment by vruano** [comment]
The second commit is just to fix another standing issue in master (testing), not really part of this pull-request but spotted and fixed while doing work in it.

**Comment by vruano** [comment]
Example syntax:

java -jar hellbender-protected.jar ConvertGSVariantsToSegments -V gs-calls.vcf.gz -O my-output.tab -targets my-targets.tab

**Comment by vruano** [comment]
Just rebased it, to see whether the testing problems go away.

**Comment by vruano** [comment]
@kareembroad did you have a chance to review the PR?

**Comment by vruano** [comment]
@kareembroad, beside the travis failure do you have any comments on the code; is there anything that you rather change.

**Comment by vruano** [comment]
Ok I seem to been able to fix the Travis problem. The coverage is worse mainly because I put some of the test classes under the main src directory, once I move them to test, that should be fixed as well. @kareembroad, any comments on the code?

**Comment by vruano** [comment]
Done some commit squashing and moved the integration test to the test source folder.

**Comment by vruano** [comment]
No more changes unless there some review comments to address.
Is there any @kareembroad?

**Comment by vruano** [comment]
@kareembroad. Can you take a look at the pull request today?

**Comment by vruano** [comment]
@LeeTL1220 can you take a look into this pull-request? Second commit is a piggybacking fix for a existing problem in master test.

File: scripts/install_R_packages.R [hunk 1]
```diff
@@ -13,4 +13,9 @@ dependencies = c("naturalsort")
 if (!all(dependencies %in% rownames(installed.packages()))) {
   install.packages(setdiff(dependencies, rownames(installed.packages())), repos="http://cran.cnr.Berkeley.edu")
 }
+
+hmmUrl = "http://cran.r-project.org/src/contrib/HMM_1.0.tar.gz"
```
Comment (LeeTL1220) [inline scripts/install_R_packages.R]: This is for testing only, right?  We are not going to be dependent on another R package, right?

File: src/main/java/org/broadinstitute/hellbender/tools/exome/eval/ConvertGSVariantsToSegments.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Tool to convert Genotype Strip Variant call file into {@link CopyNumberTriStateSegmentRecord} table file.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        summary = "Convert GS VCF file contents into a CopyNumberTriState segment file",
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/eval/ConvertGSVariantsToSegments.java]: Can you spell out Genome STRiP  and include a URL?
http://www.broadinstitute.org/software/genomestrip/ should work

File: src/main/java/org/broadinstitute/hellbender/tools/exome/eval/ConvertGSVariantsToSegments.java [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        oneLineSummary = "Convert GS VCF file into a segment file",
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/eval/ConvertGSVariantsToSegments.java]: Only spell out genome strip for the one line summary.  No need for the URL

**Comment by LeeTL1220** [comment]
@vruano Why is the R library needed?  Apologies if I missed something obvious.

File: scripts/install_R_packages.R (hunk 1 above)
Comment (vruano) [inline scripts/install_R_packages.R]: Yup, just for testing.

**Comment by vruano** [comment]
To verify that our HMM engine is doing the right comparing results with another implementation, in this case one available in R.

**Comment by vruano** [comment]
I have moved the additional R package installation to a separate script.

**Comment by vruano** [comment]
Back to @LeeTL1220

File: .travis.yml [hunk 4]
```diff
@@ -48,6 +48,7 @@ before_install:
 - sudo apt-get install -y --force-yes r-base-dev=3.1.3-1trusty
 - sudo apt-get install -y --force-yes r-base-core=3.1.3-1trusty
 - sudo Rscript scripts/install_R_packages.R
+- sudo Rscript scripts/install_R_packages_for_testing.R
```
Comment (LeeTL1220) [inline .travis.yml]: There is documentation (on the forum and in README) that will need to be updated to match this step. 

Also, you should update the Dockerfile.

**Comment by vruano** [comment]
I merged those scripts back again as discussed.

**Comment by vruano** [comment]
Back to @LeeTL1220

**Pull request merged**